### ISSUE TITLE
Open containers should spill everywhere if in storage on slipping

### ISSUE THREAD

**Issue by metalgearsloth** [space-wizards/space-station-14#17519]
Oh no your 10 large beakers of pre-mix chems got all over the floor :(

**Comment by CrigCrag** [comment]
if this happens then closing flasks should be a thing you can do (detective flask .etc)

**Comment by Matz05** [comment]
Yeah, more consequences to carrying open liquid containers are a good thing. The ability to reclose bottles like the detective's flask or screwtop bottles would be good, too.

**Comment by Myakot** [comment]
Is stam crit considered slipping? Imagine just blowing up to heavens when you get battoned.
Also would a banana pie throw do the same? (since it "slips" you even if you're not moving)

**Comment by Geekyhobo** [comment]
> Yeah, more consequences to carrying open liquid containers are a good thing. The ability to reclose bottles like the detective's flask or screwtop bottles would be good, too.

I think this would also be a good idea for really any container with stuff, giving them open and closed states, ie, chem has a open pill bottle on them and they spill and go everywhere when they trip. Or take the mixed box of lights, trip and break all the lights, but as a start, beakers and stuff like the flask should be an easy task.

**Comment by SlamBamActionman** [comment]
Tons of bottles have closable caps now, makes sense
